README.md

```diff
@@ -62,7 +62,18 @@ bundle exec rspec spec/models
 
 # Run only specs for AccountsController
 bundle exec rspec spec/controllers/accounts_controller_spec.rb
+
+# Run only specs on a particular line. e.g line 14
+bundle exec rspec spec/controllers/accounts_controller_spec.rb:14
+
+# Run only specs that are tagged slow 
+bundle exec rspec spec/controllers/accounts_controller_spec.rb --tag slow 
+
+# Skip all specs that are tagged slow 
+bundle exec rspec spec/controllers/accounts_controller_spec.rb --tag "~slow"
+
 ```
+You can find out more about rspec tags [here.](https://relishapp.com/rspec/rspec-core/v/2-3/docs/command-line/tag-option)
```

Comment:
Rather than doing this (and linking to an unsupported version of the docs) can you link to the documentation about how to run different combination of specs, then you can eliminate the arbitrary examples above.